### PR TITLE
(maint) Add stale cleanup

### PR DESCRIPTION
**Pull request by pauby**
## Description Of Changes
Added cleanup of stale issues and PR's with the '0 - Waiting On User' label..

## Motivation and Context
This will make it easier to manage the repository as stale issues will auotmatically be labelled, commented on and then closed if no follow-up work is done.

## Testing
N/A

### Operating Systems Testing
N/A

## Change Types Made
* [ ] Bug fix (non-breaking change).
* [ ] Feature / Enhancement (non-breaking change).
* [ ] Breaking change (fix or feature that could cause existing functionality to change).
* [ ] Documentation changes.
* [ ] PowerShell code changes.

## Change Checklist
* [ ] Requires a change to the documentation.
* [ ] Documentation has been updated.
* [ ] Tests to cover my changes, have been added.
* [ ] All new and existing tests passed?
* [ ] PowerShell code changes: PowerShell v2 compatibility checked?

### CODE REVIEW

**Review by AdmiringWorm — APPROVED** [review]
The changes themself look good to me, not sure if we should ensure that all issues we want to keep are correctly labeled before pulling in this GitHub action though.

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      - uses: actions/stale@v4.1.0
+        id: stale
+        with:
+          days-before-stale: 30
+          days-before-close: 14
+          exempt-all-assignees: false
+          exempt-draft-pr: true
+          stale-issue-label: "Pending Closure"
+          stale-pr-label: '0 - Waiting on User'
+          only-pr-labels: '0 - Waiting on User'
```
Comment (AdmiringWorm) [inline .github/workflows/stale.yml:24]: Based on your last comment, if the intention is that only issues **AND** PR's that have been labeled with the `0 - Waiting on User` should be considered to be stale,

then this value should be:

```suggestion
          only-labels: '0 - Waiting on User'

```

See Readme for the stale action: https://github.com/actions/stale#only-labels

**Review by AdmiringWorm — CHANGES_REQUESTED** [review]
After my misunderstanding of the intention was corrected, there is one change needed. It would also be good if you are able to rebase the PR.

**Review by gep13 — APPROVED** [review]
LGTM!

**Comment by gep13** [comment]
@pauby thanks for getting this added.  This won't be active until we get to the 2.0.0 release, since the workflow won't be on the develop branch until then, but I think that this is a good time to start making use of this as well.

**Pull request merged**